packages/create-turbo/src/index.ts

```diff
@@ -178,69 +192,46 @@ async function run() {
     };
   });
 
-  sharedPkg.packageManager = `${
-    answers.packageManager.command
-  }@${getPackageManagerVersion(answers.packageManager)}`;
+  sharedPkg.packageManager = `${packageManager.command}@${packageManagerVersion}`;
   sharedPkg.name = projectName;
 
   // write package.json
-  await fse.writeFile(
+  fs.writeFileSync(
     path.join(projectDir, "package.json"),
     JSON.stringify(sharedPkg, null, 2)
   );
 
-  if (flags.install) {
-    console.log();
-    console.log(`>>> Creating a new turborepo with the following:`);
-    console.log();
-    console.log(` - ${chalk.bold("apps/web")}: Next.js with TypeScript`);
-    console.log(` - ${chalk.bold("apps/docs")}: Next.js with TypeScript`);
-    console.log(
-      ` - ${chalk.bold("packages/ui")}: Shared React component library`
-    );
-    console.log(
-      ` - ${chalk.bold(
-        "packages/eslint-config-custom"
-      )}: Shared configuration (ESLint)`
-    );
-    console.log(
-      ` - ${chalk.bold(
-        "packages/tsconfig"
-      )}: Shared TypeScript \`tsconfig.json\``
-    );
-    console.log();
+  console.log();
+  console.log(`>>> Created a new turborepo with the following:`);
+  console.log();
```

Comment:
Could also append some `\n` to the end of message above, instead of blank lines, but not at all blocking

Reply:
This was just moved out of an `if` I intentionally didn't modify it at all.